src/main/java/com/github/osisoft/ocs_sample_library_preview/security/RolesClient.java

```diff
@@ -0,0 +1,64 @@
+package com.github.osisoft.ocs_sample_library_preview.security;
+
+import java.io.*;
+import java.net.*;
+import java.util.ArrayList;
+
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+
+import com.github.osisoft.ocs_sample_library_preview.BaseClient;
+import com.github.osisoft.ocs_sample_library_preview.SdsError;
+
+public class RolesClient {
+    private String baseUrl = null;
+    private String apiVersion = null;
+    private Gson mGson = null;
+    private BaseClient baseClient;
+    // REST API url strings
+    // base of all requests
+    private String requestBase = "api/{apiVersion}/Tenants/{tenantId}";
+    private String rolesBase = requestBase + "/Roles";
+
+    public String CommunityMember = "f79a55da-7c76-4600-a809-0f62ca9971d9";
```

Comment:
same comments as the python library. whatever we end up doing there, we should do here.

Reply:
Made this consistent with the Python lib updates